Fold GFSv16.1.6 updates into GFSv16.2.0 package

KateFriedman-NOAA opened the pull request:
**Description**

This PR folds in the `GFSv16.1.6` DO-4 updates (Issue #656) into the `GFSv16.2.0` package. The GSI updates are already incorporated in the `gfsda.v16.2.0` tag.

- add `docs/Release_Notes.gfs.v16.1.6.txt`
- update `docs/Release_Notes.gfs.v16.2.0.md` to increment prior GFS version number from `v16.1.5` to `v16.1.6` since `v16.1.6` will technically go in before `v16.2.0`.
- update `config.anal` to add new historical fix file if-block for dates prior to the `v16.1.6` implementation

Additionally:

1. update `Externals.cfg` with `GFSv16.2.0` component version tags already set in `checkout.sh`
2. update `BASE_GIT`, `BASE_SVN`, and `DMPDIR` dev paths for WCOSS2 in the `setup_expt*` scripts; now point to emc.global group account space and not the initial personal account space set up during testing before emc.global was established

Refs: #399, #662

CatherineThomas-NOAA committed:
GFS v16.1.6: GSI update to add commercial GPSRO in DO-4

Updates GSI tag from gfsda.v16.1.5 to gfsda.v16.1.6. Delivery
Order 4 (DO-4) of NOAA's commercial GPS-RO data buy includes
occultations from both Spire and GeoOptics.  GeoOptics was
previously assimilated in DO-2 and Spire is assimilated
currently in DO-3.  The update for DO-4 is a single line
change in the global_convinfo.txt fix file to turn the
GeoOptics data assimilation back on.  Updates were also made
to include a new historical fix file and update documentation.

Refs: #656

CatherineThomas-NOAA committed:
GFS v16.1.6: Update release notes and comment in config.anal

Fixed typo in release notes.  Updated the commented out block
in config.anal which has the next needed historical convinfo
file.

Refs: #656

KateFriedman-NOAA committed:
Update release notes for current ops version

- increment the current operational GFS version in the release notes
from v16.1.5 to v16.1.6
- the DO-4 v16.1.6 upgrade is happening before v16.2.0 and was also
folded into the v16.2.0 package (GSI component)

Refs: #399, #656

KateFriedman-NOAA committed:
Merge remote-tracking branch 'upstream/release/gfs.v16.1.6' into feature/ops-wcoss2

* upstream/release/gfs.v16.1.6:
  GFS v16.1.6: Update release notes and comment in config.anal
  GFS v16.1.6: GSI update to add commercial GPSRO in DO-4

KateFriedman-NOAA committed:
Update Externals.cfg with GFSv16.2.0 component versions

Refs: #399

CatherineThomas-NOAA reviewed (approved):
Looks good from the v16.1.6 perspective.

KateFriedman-NOAA committed:
Update DMPDIR and BASE_GIT paths for WCOSS2

- update the BASE_GIT and DMPDIR paths for WCOSS2 in the setup scripts
to move them to the emc.global group account area and away from the
initially established folders in a personal account space

Refs: #399, #662
